### PR TITLE
Add multiview example

### PR DESCRIPTION
**Pull request by fe1ixz**
This PR adds the multiview sample with projection layer. @toji

### CODE REVIEW

**Review by toji — CHANGES_REQUESTED** [review]
Only a couple of minor points of feedback on this one, thanks!

File: layers-samples/proj-multiview.html [hunk 1]
```diff
@@ -0,0 +1,291 @@
+<!doctype html>
+<!--
+Copyright 2018 The Immersive Web Community Group
```
[the remark this answers is not in the document]
Comment (fe1ixz) [reply layers-samples/proj-multiview.html]: Changed to 2021

File: layers-samples/proj-multiview.html [hunk 2]
```diff
@@ -0,0 +1,291 @@
...
+      session.end();
+    }
+
+    function onSessionEnded(event) {
+      if (event.session.isImmersive) {
+        xrButton.setSession(null);
+      }
+    }
+
+    function create_depth_texture(width, height) {
```
[the remark this answers is not in the document]
Comment (fe1ixz) [reply layers-samples/proj-multiview.html]: Removed function

File: layers-samples/proj-multiview.html [hunk 3]
```diff
@@ -0,0 +1,291 @@
...
+
+        let views = [];
+        for (let view of pose.views) {
+          let viewport = null;
+          if (session.isImmersive) {
+            glLayer = xrGLFactory.getViewSubImage(session.renderState.layers[0], view);
+            viewport = glLayer.viewport;
+            glLayer.framebuffer = xrFramebuffer;
+            gl.bindFramebuffer(gl.FRAMEBUFFER, xrFramebuffer);
+            if (views.length == 0) { // for mutliview we need to set fbo only once
```
[the remark this answers is not in the document]
Comment (fe1ixz) [reply layers-samples/proj-multiview.html]: Corrected typo

File: layers-samples/proj-multiview.html [hunk 4]
```diff
@@ -0,0 +1,291 @@
...
+        webgl2: true
+      });
+      document.body.appendChild(gl.canvas);
+
+      samples = gl.getParameter(gl.MAX_SAMPLES);
+
+      mv_ext = gl.getExtension('OCULUS_multiview');
+      if (mv_ext) {
+        console.log("OCULUS_multiview extension is supported");
+        is_multiview = true;
```
[the remark this answers is not in the document]
Comment (fe1ixz) [reply layers-samples/proj-multiview.html]: Removed is_multiview, and added multiview compatibility check before user initiate immersive session, showing multiview compatibility status and disabling enter vr button if not compatible
Screen Shot 2021-07-07 at 5 38 03 PM

**Comment by toji** [comment]
Thank you! As with the other sample I need to squash this one to merge it, but I don't think that will cause any problems.

**Pull request merged**